erica/domain/Shared/tax_id_number.py

```diff
@@ -0,0 +1,35 @@
+from collections import Counter
+from stdnum.iso7064.mod_11_10 import is_valid
+
+
+class TaxIdNumber(str):
+
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate_tax_id_number
+
+    @classmethod
+    def validate_tax_id_number(cls, tax_id_number: str) -> str:
```

Comment:
I suppose you just cpoied this from steuerlotse and I don't have to check that the function works correctly?

Reply:
Yes, I copied that from steuerlotse and adapted it to make it work with pydantic instead of WTForms. I also copied the tests